desktop/cmp/dash/container/DashContainerModel.ts

```diff
@@ -214,8 +214,12 @@ export class DashContainerModel
         // Initialize GoldenLayout with initial state once ref is ready
         this.addReaction(
             {
-                track: () => [this.containerRef.current, this.layoutLocked],
-                run: () => this.loadStateAsync(this.state)
+                track: () => [this.containerRef.current, this.layoutLocked] as const,
```

Comment:
I did not test the layoutLocked change path, but good to ensure that all working as expected